tcf_website/views/team_info.json

```diff
@@ -148,13 +148,6 @@
       "class": "2021",
       "img_filename": "ENG_James_Lim.jpg",
       "github": "yl2vv"
-    },
-    {
-      "name": "Kartik Chugh",
-      "role": "Developer",
-      "class": "2023",
-      "img_filename": "ENG_Kartik_Chugh.jpg",
-      "github": "KartikChugh"
```

Comment:
I think you accidentally deleted Kartik lol might wanna fix that

Reply:
oh yeah, I'll definitely fixed that "accident"